src/Database/IPFS/Methods.ts

```diff
@@ -19,6 +19,11 @@ async function findOne(
   if (document === null) {
     return undefined;
   }
-  delete (document as any)._id;
+
+  if ("_id" in document) {
+    const { _id, ...doc } = document;
+    return doc;
+  }
+
   return document;
```

Comment:
I think the delete is fine in this instance unless there is a specific concern? Avoid destructured copy in favor of mutation of the original object. Since we aren't worried about the mutation here breaking expectations 🤔 

Apart from the obvious `any` casting 😸

Reply:
The delete operator should be avoided because it can prevent some optimizations of JavaScript engines.
https://docs.rome.tools/lint/rules/nodelete

Reply:
I can instead add a rule ignore comment!

Reply:
If its a performance issue then its a good change 👍🏻